Tools/TestWebKitAPI/Tests/WebKitCocoa/CreateWebArchive.mm

```diff
@@ -1385,6 +1385,73 @@ function onImageLoad() {
     Util::run(&saved);
 }
 
+static const char* htmlDataBytesForEmptyStyleSheet = R"TESTRESOURCE(
+<head>
+<link rel="stylesheet" href="empty.css">
+</head>
+<script>
+window.webkit.messageHandlers.testHandler.postMessage("done");
+</script>
+)TESTRESOURCE";
+
+TEST(WebArchive, SaveResourcesEmptyStyleSheet)
+{
+    RetainPtr<NSURL> directoryURL = [NSURL fileURLWithPath:[NSTemporaryDirectory() stringByAppendingPathComponent:@"SaveResourcesTest"] isDirectory:YES];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    [fileManager removeItemAtURL:directoryURL.get() error:nil];
+
+    auto configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
+    auto schemeHandler = adoptNS([[TestURLSchemeHandler alloc] init]);
+    [configuration setURLSchemeHandler:schemeHandler.get() forURLScheme:@"webarchivetest"];
+    NSData *htmlData = [NSData dataWithBytes:htmlDataBytesForEmptyStyleSheet length:strlen(htmlDataBytesForEmptyStyleSheet)];
+    NSData *cssData = [[NSData alloc] init];
```

Comment:
Seems like we should adopt `[[NSData alloc] init]`, no? Seems like this may be leaking.